### ISSUE TITLE
Bug report: Space Charater at the end of a foldername

### ISSUE THREAD

**Issue by X-eno** [localsend/localsend#1503]
### Describe the bug

If a foldername contains a space at the end it wont be tranfered to the target. 
example: "Pictures " instead of "Pictures".
It skips the folder with the notice "cant find folder" (sorry can't remember the exact notice).
Maybe it's nothing special and won't happen with many because it's uncommen that naming. Just happen to notice it.
From Android 11 device to Windows PC.

### To reproduce

1. Go to a folder add a space at the end
2. Try to tranfer it

### Expected behavior

Well just transfer it even with the space

### Desktop (please complete the following information)

Windows 10

### Smartphone (please complete the following information)

Android 11

**Comment by Mujtaba0150** [comment]
I don't think this should even be possible because Android (Samsung at least) doesn't let you name folders with a space at the end of the folder name

**Comment by X-eno** [comment]
I got a Oneplus 6 and Oneplus 6T here and i could name the folder with a space at the end.
